nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ReplaceText.java

```diff
@@ -79,7 +80,8 @@
 @SystemResourceConsideration(resource = SystemResource.MEMORY)
 public class ReplaceText extends AbstractProcessor {
 
-    private static Pattern REPLACEMENT_NORMALIZATION_PATTERN = Pattern.compile("(\\$\\D)");
+    private static Pattern QUOTED_GROUP_REF_PATTERN = Pattern.compile("('\\$\\d+')");
+    private static Pattern LITERAL_QUOTED_PATTERN = Pattern.compile("literal\\(('.*?')\\)",Pattern.DOTALL);
```

Comment:
Expression Language allows single quotes and double quotes to be used interchangeably. Can this pattern be updated to handle both?

Reply:
done